scripts/ariitk_create_lib

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
```

Comment:
change shebangs

Reply:
this shebang works well for zsh , please test script on your zsh terminal